### DISCUSSION TITLE
Independent delay primary and secondary subtitles

### DISCUSSION THREAD

**Discussion by simepy** [smplayer-dev/smplayer#395] [Q&A] [answered]
Hello great community of SMPlayer

I would like to know if is possible to + / - the delay of the primary and / or secondary subtitles independently from each other ?

If the feature is not available:
  - It's will be a great feature to add I think
  - Do you know any workaround ?

Thank you in advance

**Comment by smplayer-dev** [comment, marked as answer]
I'm afraid this is not possible because mpv only has one option (`-sub-delay`) which affects both subtitles.
It seems there was a pull request to implement a `--secondary-sub-delay` option but the mpv developers rejected it.

**Reply by simepy** [reply]
Hi @smplayer-dev thanks for your answer.

Ok, I see the issue..., about mPlayer ?

**Reply by smplayer-dev** [reply]
mplayer doesn't even support displaying two subtitles at the same time.

**Comment by gregorko** [comment]
It looks like the feature is available in mpv now. Would be nice to also see this in smplayer.
